packages/frontend/src/hooks/pivotTable/pivotQueryResults.ts

```diff
@@ -116,20 +117,38 @@ export const pivotQueryResults = ({
 
         for (let nMetric = 0; nMetric < metrics.length; nMetric++) {
             const metric = metrics[nMetric];
-            const indexRowValues = [
-                ...indexDimensions.map((d) => row[d].value),
-                ...(pivotConfig.metricsAsRows ? [metric] : []),
-            ];
-            const headerRowValues = [
-                ...headerDimensions.map((d) => row[d].value),
-                ...(pivotConfig.metricsAsRows ? [] : [metric]),
-            ];
+
+            const indexRowValues: PivotValue[] = indexDimensions
```

Comment:
why did this change to map + concat?

Reply:
change for typing because it was not determining types with spread properly